app/data_model/answer_store.py

```diff
@@ -1,7 +1,12 @@
 import re
+
+from datetime import datetime
+from flask import g
```

Comment:
We should avoid flask globals being used outside of the views and globals.py - this only appears to be used to get the schema json, can this be passed to the answer store on initialisation?